pages/reference/OS/configuration.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Configuration
+excerpt: Instructions for editing and customizing config.json for modifying the behavior of {{ $names.os.lower }}
+---
+
+# Configuring {{ $names.os.lower }}
+
+The behavior of {{ $names.os.lower }} can be configured by editing the `config.json` file. This file is located in the [boot partition][boot-partition] and accepts a [range of fields](#valid-fields) to modify the behavior of the host OS.
+
+**Note:** The `config.json` file is different from the [`config.txt`][config-txt] file, also located in the boot partition, which is used by the Raspberry Pi to set device configuration options.
+
+Before the device is [provisioned][provisioned], you may edit `config.json` by mounting a flashed SD card (which will mount as `resin-boot`) and editing the file directly. The boot partition is mounted on the device at `/mnt/boot`, and so on the device, the file is located at `/mnt/boot/config.json`. For example, to output a formatted version of `config.json` on a device, use the following commands:
+
+```shell
+{{ $names.company.lower }} ssh <uuid>
+cat /mnt/boot/config.json | jq '.'
+```
+
+**Warning:** Editing config.json on a provisioned device should be done very carefully as any mistakes to this file can leave a device inaccessible.
```

Comment:
nit: "mistakes to this file" doesn't sound right. Perhaps "any mistakes made in the syntax of this file" or "any errors created in this file" ?